src/Apps/CollectorProfile/Routes/Saves2/Components/SavesItem.tsx

```diff
@@ -0,0 +1,74 @@
+import { Box, Flex, Text } from "@artsy/palette"
+import { FourUpImageLayout } from "Apps/CollectorProfile/Routes/Saves2/Components/FourUpImageLayout"
+import { StackedImageLayout } from "Apps/CollectorProfile/Routes/Saves2/Components/StackedImageLayout"
+import { FC } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { extractNodes } from "Utils/extractNodes"
+import { SavesItem_item$data } from "__generated__/SavesItem_item.graphql"
+
+const ITEM_DROP_SHADOW = "0px 2px 12px rgba(0, 0, 0, 0.13)"
```

Comment:
Use one of our shadows or get design to alter it: https://github.com/artsy/palette/blob/main/packages/palette/src/helpers/shadow.ts

Reply:
ok, I'll clarify this point with the designer